### PR TITLE
2.6.1.5 release notes

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

File: content/reference/release-notes/portworx.md [hunk 1]
```diff
@@ -6,6 +6,20 @@ keywords: portworx, release notes
...
+
+### Fixes
+
+The following issues have been fixed:
+
+|**Issue Number**|**Issue Description**|
+|----|----|
+| PWX-16796 | Proxy username and password were ignored as part of PX_HTTP_PROXY on essentials causing license renewal to fail.<br/><br/>**User Impact:** Px-essentials cluster will go into license expired mode with PX_HTTP_PROXY<br/><br/>**Resolution:** PX-essentials now honors Username and Password fields given as part of PX_HTTP_PROXY to successfully make connections with proxy |
+| PWX-16775 | Reduce ListObject timings with google object-store. This resulted in cloudsnaps commands timing out when the bucket involved in cloudsnaps had lot of objects. |
+| PWX-16432 | Multipathd configuration files were not correctly set for blacklisting px devices<br/><br/>**User Impact:** Incorrect entries in Multipathd conf file caused other devices to be not handled correctly on the host<br/><br/>**Resolution:** Disable handling the Multipathd conf file by default and give an option to user to add blacklist entries to Multipathd conf file correctly through runc install argument " -enable-mpcfg-update" |
```
Comment (vilasd) [inline content/reference/release-notes/portworx.md:21]: We don't need it as we document in 2.6.1.4

Can you add bold letter note on the Top saying Added Support for OCP 4.6.1

File: content/reference/release-notes/portworx.md [hunk 2]
```diff
@@ -6,6 +6,20 @@ keywords: portworx, release notes
...
+November 16, 2020
+
+### Fixes
+
+The following issues have been fixed:
+
+|**Issue Number**|**Issue Description**|
+|----|----|
+| PWX-16796 | Proxy username and password were ignored as part of PX_HTTP_PROXY on essentials causing license renewal to fail.<br/><br/>**User Impact:** Px-essentials cluster will go into license expired mode with PX_HTTP_PROXY<br/><br/>**Resolution:** PX-essentials now honors Username and Password fields given as part of PX_HTTP_PROXY to successfully make connections with proxy |
+| PWX-16775 | Reduce ListObject timings with google object-store. This resulted in cloudsnaps commands timing out when the bucket involved in cloudsnaps had lot of objects. |
```
Comment (vilasd) [inline content/reference/release-notes/portworx.md:20]: @talakad can you make this RL into the format of other RL
Issue Description
User Impact
Resolution

Comment (talakad) [reply content/reference/release-notes/portworx.md:24]: For PWX-16775:
Issue description:  Google object-store did not have pagination for object enumeration and any list call ended up listing everything in the bucket. 
User Imapct: Cloudsnap backup and restores  failed to start with request timing  out. Even listing cloudsnaps through pxctl would timeout.
Resolution: Add pagination to object enumeration with google objectstore.

**Review by vilasd — APPROVED** [review]
LGtm

**Pull request merged**